src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.mssql_api_is_not_changed.approved.txt

```diff
@@ -260,7 +260,7 @@ Microsoft.DotNet.Interactive.SqlServer
     protected System.Boolean CanDeclareVariable(System.String name, System.Object value, ref System.String& msg)
     public System.Threading.Tasks.Task ConnectAsync()
     protected System.String CreateVariableDeclaration(System.String name, System.Object value)
-    protected System.Collections.Generic.Dictionary<System.String,System.Collections.Generic.IReadOnlyCollection<Microsoft.DotNet.Interactive.Formatting.TabularData.TabularDataResource>> get_QueryResults()
+    protected System.Collections.Generic.Dictionary<System.String,System.Object> get_Variables()
```

Comment:
Exposing these as dictionaries mean they can be directly updated programmatically, bypassing other ways of declaring a variable. I'm not sure we want that.

Reply:
good point, removed access and added a StoreQueryResult method (protected)